resteasy-core/src/main/java/org/jboss/resteasy/mock/MockHttpRequest.java

```diff
@@ -122,6 +122,13 @@ public static MockHttpRequest put(String uri) throws URISyntaxException
       request.httpMethod = "PUT";
       return request;
    }
+   
```

Comment:
```suggestion

```
@massimoadriaens According to [this message](https://travis-ci.org/github/resteasy/Resteasy/builds/750855092#L2935), it seems to be due to the trailing spaces

Reply:
thanks!
How did you find it in that HUGE logging?

Reply:
@massimoadriaens I'm simply used to it :-)